### PR TITLE
ROCANA-3409: Kite should include the rocana maven repo

### PR DESCRIPTION
**Pull request by joey**
- This also moves the maven plugin to only the CDH profile.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1353,15 +1363,15 @@
...
           <name>hadoop.profile</name>
           <value>cdh5</value>
         </property>
       </activation>
-      <properties>
-        <!-- test cdh5 with the same jars that were produced for default profile -->
-        <maven.main.skip>true</maven.main.skip>
-      </properties>
+      <modules>
+        <module>kite-maven-plugin</module>
```
Comment (bradcupit) [inline pom.xml:1377]: Will we ever need this module for HDP?
I assume no, but perhaps this setup is unusual enough to justify a quick line of documentation?

File: pom.xml [hunk 2]
```diff
@@ -1353,15 +1363,15 @@
...
       <activation>
+        <activeByDefault>true</activeByDefault>
         <property>
           <name>hadoop.profile</name>
           <value>cdh5</value>
         </property>
       </activation>
-      <properties>
-        <!-- test cdh5 with the same jars that were produced for default profile -->
-        <maven.main.skip>true</maven.main.skip>
```
Comment (bradcupit) [inline pom.xml:1363]: I must confess I don't understand this part.

**Comment by bradcupit** [comment]
Other than the two questions (which are probably more for my benefit than the PR's), +1

File: pom.xml (hunk 1 above)
Comment (joey) [inline pom.xml:1377]: We don't use this module at all. I thought about pulling it out entirely, but this seemed like a good middle ground. I'll add a comment.

File: pom.xml (hunk 2 above)
Comment (joey) [inline pom.xml:1363]: This has to do with how upstream Kite uses profiles. There, they always build against one set of dependencies and their profiles are only used for running unit/integration tests. That's why there weren't any code changes needed to support HDP, but doing specific builds is a lot less error prone.

**Pull request merged**